sql/database.sql

```diff
@@ -5446,9 +5475,15 @@ CREATE TABLE `patient_data` (
   `guardianphone` TEXT,
   `guardianworkphone` TEXT,
   `guardianemail` TEXT,
+  `sexual_orientation` varchar(255) DEFAULT NULL,
+  `gender_identity` varchar(255) DEFAULT NULL,
```

Comment:
use TEXT for both of these
(this is a special table where we use TEXT rather than varchar to avoid breaking size limit; historical note, in past we used all varchar(255) and things broke when users tried to add to many entries/columns to this table in the layout editor, so we then changed them to text)